src/components/Footer/Footer.tsx

```diff
@@ -0,0 +1,40 @@
+import IconFacebook from '../../assets/images/icons/facebook.svg';
+import IconInstagram from '../../assets/images/icons/instagram.svg';
+import IconLinkedin from '../../assets/images/icons/linkedin.svg';
+import { SocialIcon } from '../ui/SocialIcon/SocialIcon';
+
+import styles from './footer.module.css';
+
+function Footer() {
+    return (
+        <footer className={styles.footer}>
+            <div className={styles.footer__container}>
+                <ul className={styles.footer__socList}>
+                    <li className={styles.footer__socItem}>
+                        <SocialIcon url="https://www.facebook.com/" iconPath={IconFacebook} />
```

Comment:
As an optional addition enhancement, you could make a constant `SOCIAL_ICONS: SocialIconProps[]` and map it here.

Reply:
Very good suggestion!
Fixed